src/Containers/MassTransit.AspNetCoreIntegration/ServiceCollectionExtensions.cs

```diff
@@ -105,6 +95,16 @@ public static class ServiceCollectionExtensions
             return services;
         }
 
+        static void ConfigureLogging(IServiceProvider provider, ILoggerFactory loggerFactory = null)
+        {
+            if (loggerFactory == null)
+                loggerFactory = provider.GetService<ILoggerFactory>();
+            var diagnosticSource = provider.GetService<DiagnosticSource>();
```

Comment:
it is not the best way to do this, you could have multiple diagnostics sources, it's better to provide it explicitly using for example `host settings`

Reply:
Well, these methods weren't created by me, and I wasn't doing to add parameters to them without getting some feedback on the best approach. @alexeyzimarev created them, so his input is needed.